### PR TITLE
docs: update the Angular roadmap as of May 2022

### PR DESCRIPTION
**Pull request by mgechev**
* Update project completion status
* Update project description based on state
* Add new projects to the future section

### CODE REVIEW

File: aio/content/guide/roadmap.md [hunk 1]
```diff
@@ -14,37 +14,29 @@ For example, features are released in the next minor after they are complete, or
```
Comment (AndrewKushnir) [inline aio/content/guide/roadmap.md:14]: @mgechev nit: you could also change the "Last updated" date above as well?

**Review by jelbourn — APPROVED** [review]
LGTM

File: aio/content/guide/roadmap.md [hunk 2]
```diff
@@ -14,37 +14,43 @@ For example, features are released in the next minor after they are complete, or
...
-We are first testing the solution with more than 2,500 projects at Google to ensure a smooth migration path for the external community.
-
 ### Implement APIs for optional NgModules
 
-In the process of making Angular simpler, we are working on introducing APIs that allow developers to initialize applications, instantiate components, and use the router without NgModules.
-We will share an RFC to start a design discussion with the community, followed by a series of pull request with the implementation.
-In the first part of this project we finalized the design of standalone components, directives, and pipes \([RFC](https://github.com/angular/angular/discussions/43784)\).
+In the process of making Angular simpler, we are working on introducing APIs that allow developers to initialize applications, instantiate components, and use the router without NgModules. Angular v14 introduces developer preview of the APIs for standalone components, directives, and pipes. In the next few quarters we'll collect feedback from developers and finalize the project making the APIs stable. As the next step we will work on improving use cases such ase `TestBed`, Angular elements, etc.
```
Comment (Adonsio) [inline aio/content/guide/roadmap.md:19]:
```suggestion
In the process of making Angular simpler, we are working on introducing APIs that allow developers to initialize applications, instantiate components, and use the router without NgModules. Angular v14 introduces developer preview of the APIs for standalone components, directives, and pipes. In the next few quarters we'll collect feedback from developers and finalize the project making the APIs stable. As the next step we will work on improving use cases such as `TestBed`, Angular elements, etc.
```

little typo in there

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 8db9930aa01f0d8a3aaf3b827aba648b4d19ee0b.

**Pull request closed without merging**